sdk/core/azure-core/src/main/java/com/azure/core/implementation/annotation/Fluent.java

```diff
@@ -10,7 +10,8 @@
 import static java.lang.annotation.ElementType.TYPE;
 
 /**
- * Annotation given to all classes that are expected to provide a fluent API to end users. If a class has this annotation,
+ * Annotation given to all classes that are expected to provide a fluent API to end users. If a class has this
+ * annotation,
```

Comment:
Weird line break.

Reply:
i think this is still weired line break. Do i miss something?